packages/frontend/components/composed/journal/JournalNavBar/JournalNavBar.styles.ts

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+import { pxToRem } from "@wdp/lib/theme/functions";
+
+export const Nav = styled.div`
+  padding-block: ${pxToRem(36)};
+  min-height: ${pxToRem(104)};
+`;
+
+export const NavInner = styled.div`
+  --button-secondary-bg-color: var(--color-custom10);
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const LeftSide = styled.div``;
+
+export const RightSide = styled.div`
+  max-width: ${pxToRem(376)};
```

Comment:
The search bar only grows to the size of it's parent, so right now it's about 250px wide. If you set this to `flex: 1 1 auto`, then this div will grow if there's room, and the search bar will be the right width on desktop 👍